src/main/java/com/spartronics4915/frc2020/Robot.java

```diff
@@ -33,8 +33,8 @@ public void robotPeriodic()
     public void disabledInit()
     {
         // TODO: verify call to DISABLED bling state
-        Logger.notice("@disabledInit: Requested BlingState.DISABLED");
-        mRobotContainer.setBlingState(BlingState.DISABLED);
+        Logger.notice("@disabledInit: Requested BlingState.BLING_COMMAND_DISABLED");
+        // mRobotContainer.setBlingState(BlingState.BLING_COMMAND_DISABLED);
```

Comment:
Any reason why this line is commented?

Reply:
I'm not sure, I didn't adjust anything in .java code, was just told to add the adjusted Robot.java into my computer and push it along with the bling code that I have.